### PR TITLE
Package unidecode.0.2.0

### PR DESCRIPTION
**Pull request by sagotch**
### `unidecode.0.2.0`
Convert unicode strings into its ASCII representation

---
* Homepage: https://github.com/geneweb/unidecode
* Source repo: git+https://github.com/geneweb/unidecode.git
* Bug tracker: https://github.com/geneweb/unidecode/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
Should this PR be closed in favour of https://github.com/ocaml/opam-repository/pull/15116

**Comment by kit-ty-kate** [comment]
If not, same request applies for this PR too

**Comment by sagotch** [comment]
This lib comes from a bigger project which, for the time being, need version 0.2.0 and not 0.3.0

File: packages/unidecode/unidecode.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+license: "GNU GPL"
+
+homepage: "https://github.com/geneweb/unidecode"
+
+bug-reports: "https://github.com/geneweb/unidecode/issues"
+
+dev-repo: "git+https://github.com/geneweb/unidecode.git"
+
+depends: [
+  "dune" { "1.10" }
```
Comment (mseri) [inline packages/unidecode/unidecode.0.2.0/opam]:
```suggestion
  "dune" { >= "1.10" }
```

If you need the specific version, you still need the `=` sign

Comment (sagotch) [reply packages/unidecode/unidecode.0.2.0/opam]: Sorry about that. Probably too early in the morning...

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**